stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewHolderFactory.kt

```diff
@@ -108,8 +112,9 @@ public open class MessageListItemViewHolderFactory {
             decoratorProvider.decorators,
             listenerContainer,
             markdown,
-            attachmentViewFactory,
-            style
+            attachmentViewHolderFactory,
+            mediaAttachmentsRecycledViewPool,
+            fileAttachmentsRecycledViewPool
```

Comment:
This doesn't look like it'd scale very well, users would have to add new RecycledViewPools for their own custom types of attachments, wouldn't they?